### ISSUE TITLE
Keyboard controls

### ISSUE THREAD

**Issue by AppPhil** [baseball435/TerraLegion#19]
For testing TerraLegion on my computer I don't like using the joystick. When I press "A" or "D" the character moves to the left or to the right. When I press space I jump only sometimes when I'm not completely standing on a block. Do you understand what I mean?

**Comment by jmrapp1** [comment]
Yeah I believe there is a small bug for the jumping which doesn't work when the space bar is pressed although it works with the joystick. It's very strange, but it needs to be looked at.

**Comment by AppPhil** [comment]
When I comment out this (https://github.com/jmrapp1/TerraLegion/blob/1030fe0be7483a22488b2a64444c0a2f0df64c04/core/src/com/jmrapp/terralegion/engine/world/PhysicsWorld.java#L175) and the line after jumping with the keyboard is working but then the player falls down very fast in some cases.

**Comment by jmrapp1** [comment]
Yeah that needs to remain. We may need to add some special case to check if the player is trying to jump and if they are, don't do "some case" in the physics logic.

**Comment by AppPhil** [comment]
Here I tried to fix it: https://github.com/AppPhil/TerraLegion/commit/1c88eb0b6b8218e13dbeb7df6d48f0bed38dffe3

With this code you can jump with "W", "UP" or "SPACE" but when there's a block over you and you collide with it you stuck there for a short time. After that you normally fall down.
